src/common/include/sirf/common/DataContainer.h

```diff
@@ -52,21 +53,55 @@ namespace sirf {
 		/// calculates the dot product of this container with another one
```

Comment:
will have to document `ptr` needs to be a `float *` (but I really don't like it...)

Reply:
Added a comment on the actual values of `void*` in `DataContainer` methods.